src/main/java/com/onionshop/UndoRedoManager.java

```diff
@@ -0,0 +1,58 @@
+package com.onionshop;
+
+import java.util.Stack;
```

Comment:
I'm not sure why this is showing up in this PR, can u pull from main and see if there are any merge conflicts?